operations/image_management/Configure_a_Remote_Build_Node.md

```diff
@@ -0,0 +1,682 @@
+# Configure a Remote Build Node
+
+Configure or create an IMS remote build node for use for image builds.
+
+* [Prerequisites](#prerequisites)
+* [Overview](#overview)
+* [Configuring a remote build node](#configuring-a-remote-build-node)
+* [Use an existing compute node](#use-an-existing-compute-node)
+* [Create a barebones IMS builder image](#create-a-barebones-ims-builder-image)
+* [Adding storage to a remote build node](#adding-storage-to-a-remote-build-node)
+* [Adding and removing an HSM lock](#adding-and-removing-an-hsm-lock)
+* [Adding remote build nodes to IMS](#adding-remote-build-nodes-to-ims)
+* [Remove remote build nodes from IMS](#remove-remote-build-nodes-from-ims)
```

Comment:
```suggestion
* [Removing remote build nodes from IMS](#removing-remote-build-nodes-from-ims)
```

Reply:
You made me change it from removing -> remove on the original PR.
Make up your mind!